src/ui/hooks/recordings.ts

```diff
@@ -303,25 +312,28 @@ export function useGetWorkspaceRecordings(
 ): { recordings: null; loading: true } | { recordings: Recording[]; loading: false } {
   const { data, error, loading } = useQuery(
     gql`
-      query GetWorkspaceRecordings($workspaceId: uuid) {
-        recordings(
-          where: { _and: { workspace_id: { _eq: $workspaceId }, deleted_at: { _is_null: true } } }
-        ) {
-          id
-          url
-          title
-          recording_id
-          recordingTitle
-          last_screen_mime_type
-          duration
-          description
-          date
-          is_private
-          user {
-            name
-            email
-            picture
+      query GetWorkspaceRecordings($workspaceId: ID!) {
```

Comment:
Interesting, I was assuming we'd do this query using `uuid` from the workspace, and `workspace(uuid: $workspaceId) {`. It does work either way though. How do you feel about the `id` vs `uuid` stuff?

Reply:
I use only one kind of ID for each table, `uuid` for recordings and `id` for everything else. I think it's a bit confusing for API clients that there are two kinds of IDs, but not a big problem.